recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd.cfg

```diff
@@ -15,6 +15,258 @@ device_tree = "/xt/domd/domd.dtb"
 # Kernel command line options
 extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd ip=192.168.1.11 rw rootwait console=hvc0 cma=64M"
 
+dt_compatible = [ "renesas,salvator-x", "renesas,r8a7796" ]
+
+dt_passthrough_nodes = [
+    "/soc",
+#clocks
+    "/extal",
+    "/extalr",
+    "/audio_clk_a",
+    "/audio_clk_b",
+    "/audio_clk_c",
+    "/can",
+    "/scif",
+    "/pcie_bus",
+    "/msiof-ref-clock",
+    "/dclkin-0",
+    "/dclkin-1",
+    "/dclkin-2",
+#other
+    "/avb-mch",
+    "/regulator0",
+    "/regulator1",
+    "/regulator-vcc-sdhi0",
+    "/regulator-vccq-sdhi0",
+    "/regulator-vcc-sdhi3",
+    "/regulator-vccq-sdhi3",
+    "/x12_clk",
+    "/audio_clkout",
+    "/sound",
+    "/vga-encoder",
+    "/vga",
+    "/vspm_if",
+    "/lvds-encoder",
+    "/lvds",
+    "/hdmi0-out",
+    "/regulator-vbus0-usb2",
```

Comment:
Please add a comment in a code explaining why we need to handle these clocks and regulators.

Reply:
I don't know, you help me understand if we need these

Reply:
removed regulators